measurements/geography_based.py

```diff
@@ -478,4 +478,52 @@ def geo_lng(dataloader, args):
 
     pickle.dump(info, open("results/{}/geo_lng.pkl".format(args.folder), "wb"))
 
+def geo_att(dataloader, args):
+    geo_att = {}
```

Comment:
slightly confusing to name a dictionary the same name as the function it's in (I know we do similar things throughout unfortunately), probably best to rename this something else